distribution/lib/Standard/Base/0.0.0-dev/src/Metadata.enso

```diff
@@ -77,3 +77,11 @@ type Widget
 
     ## Describes a file chooser.
     File_Browse label:(Nothing | Text)=Nothing display:Display=Display.When_Modified action:File_Action=File_Action.Open file_types:(Vector Pair)=[Pair.new "All Files" "*.*"]
+
+## PRIVATE
+make_single_choice : Vector -> Display -> Widget
+make_single_choice values display=Display.Always =
+    make_option value = case value of
+        _ : Vector -> Choice.Option value.first value.second
+        _ -> Choice.Option value value.pretty
```

Comment:
my personal opinion, feel free to ignore: I'd add a check here to avoid surprises.
```suggestion
        _ : Text -> Choice.Option value value.pretty
```